sqlstore/src/main/resources/db/migration_afterMigrate/05_standard.sql

```diff
@@ -934,9 +934,32 @@ JOIN BoxPosition bp
   AND bp.targetId = p.poolId;
 
 CREATE OR REPLACE VIEW PoolableElementView
-AS SELECT d.dilutionId, d.name AS dilutionName, d.concentration AS dilutionConcentration,
-  l.lowQuality AS lowQualityLibrary, p.shortName AS projectShortName, p.alias AS projectAlias
-FROM LibraryDilution d
-JOIN Library l ON l.libraryId = d.library_libraryId
-JOIN Sample s ON s.sampleId = l.sample_sampleId
-JOIN Project p ON p.projectId = s.project_projectId;
+AS SELECT
+    d.dilutionId,
+    d.name AS dilutionName,
+    d.concentration AS dilutionConcentration,
+    d.identificationBarcode AS dilutionBarcode,
+    d.lastUpdated AS lastModified,
+    l.libraryId AS libraryId,
+    l.name AS libraryName,
+    l.alias AS libraryAlias,
+    l.description AS libraryDescription,
+    l.lowQuality AS lowQualityLibrary,
+    l.platformType AS platformType,
+    s.sampleId AS sampleId,
+    s.name AS sampleName,
+    s.alias AS sampleAlias,
+    p.projectId AS projectId,
+    p.shortName AS projectShortName,
+    p.alias AS projectAlias
+  FROM LibraryDilution d
+    JOIN Library l ON l.libraryId = d.library_libraryId
+    JOIN Sample s ON s.sampleId = l.sample_sampleId
+    JOIN Project p ON p.projectId = s.project_projectId;
+
+CREATE OR REPLACE VIEW PoolableElementView_Index
+AS SELECT
+    d.dilutionId AS dilutionId,
+    i.index_indexId AS index_indexId
+  FROM LibraryDilution d
+    JOIN Library_Index i ON i.library_libraryId = d.library_libraryId;
```

Comment:
The libraryId is included in PoolableElementView, so this additional view is unnecessary. The existing Library_Index table can bridge PoolableElementView to Indices the same way it bridges Library to Indices